app/models/concerns/editable_wrapper.rb

```diff
@@ -37,7 +37,7 @@ def write_to!(instance)
     end
 
     writable_attribute_names
-      .excluding("editing_id")
+      .excluding("editing_id", "first_active_wizard_step")
```

Comment:
Similar here about putting it in `DraftVaccinationRecord#writable_attribute_names`.